static/js/components/PersonalForm.js

```diff
@@ -39,6 +39,37 @@ export default class PersonalForm extends ProfileFormFields {
     ui:                     UIState,
   };
 
+  renderRomanizedFields = () => {
+    const { profile } =  this.props;
+    const english = /^[A-Za-z]*$/;
+    let firstName = _.get(profile, ["first_name"], "");
+    let lastName = _.get(profile, ["last_name"], "");
+    let rFirstName = _.get(profile, ["romanized_first_name"], "");
+    let rLastName = _.get(profile, ["romanized_last_name"], "");
+    const showRomanizedFields = (
+      !english.test(firstName) || !english.test(lastName) || rFirstName || rLastName
```

Comment:
@rhysyngsun here if first name or last name is not in english then asking user to Romanized it or use already have Romanized first and last name